src/System.Numerics.Vectors/src/Properties/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
```

Comment:
Why is this needed? We try to stay away from InternalsVisibleTo for testing.

Reply:
HashCode has to be made internal to avoid namespace clashes in netfx, and has to be used in a strong manner due to the seed randomization.